azext_edge/edge/params.py

```diff
@@ -449,9 +449,9 @@ def load_iotops_arguments(self, _):
             arg_group="Key Vault CSI Driver",
         )
         context.argument(
-            "keyvault_sat_secret_name",
-            options_list=["--kv-sat-secret-name"],
-            help="The Key Vault secret *name* to use for the IoT Operations service account (SAT). "
+            "keyvault_spc_secret_name",
+            options_list=["--kv-spc-secret-name"],
```

Comment:
although this is public preview - some customers (and other people) could get annoyed with this change

Reply:
This change coveys a more accurate meaning of what the parameter is for. As you said, this project is classified as preview, so there is not guarantee of stability in the public surface area.